packages/squiggle-lang/src/rescript/Utility/E/E_A.res

```diff
@@ -323,32 +323,45 @@ module Floats = {
       let continuous: array<float> = []
       let discrete = FloatFloatMap.empty()
 
-      let addData = (count: int, value: float): unit => {
-        if count >= minDiscreteWeight {
-          FloatFloatMap.add(value, count->Belt.Int.toFloat, discrete)
+      // In a run of exactly minDiscreteWeigth, the first and last
+      // element indices differ by minDistance.
+      let minDistance = minDiscreteWeight - 1
+
+      let l = length(sortedArray)
```

Comment:
Maybe `len` instead of `l`? `l` can be hard to read in some fonts, easy to confuse with `1`.